### PR TITLE
Fix document selector scope

### LINKED ISSUE #8602: Duplicate type signatures/docs in hover tooltips
Sourcegraph.com/GitHub (Chrome extension)

I've seen this multiple times today, always on GitHub. No idea how to repro, and after refreshing it disappears. The first time I saw them, there were 3x, not just 2x (as you see here).

image

image

### PR DESCRIPTION
**Pull request by efritz**
Closes sourcegraph/sourcegraph#8602.

### CODE REVIEW

**Comment by efritz** [comment]
GitHub wasn't updating https://github.com/sourcegraph/code-intel-extensions/pull/264 with new commits.

cc @lguychard

File: shared/lsp/integration.test.ts [hunk 1]
```diff
@@ -216,7 +216,7 @@ describe('register()', () => {
         assert.deepStrictEqual(selector, [
             {
                 language: 'typescript',
-                pattern: 'https://sourcegraph.test/repo@rev/-/raw/**',
+                pattern: 'https://sourcegraph.test/repo@rev/-/raw/#**/**',
```
Comment (felixfbecker) [inline shared/lsp/integration.test.ts]: This looks wrong, neither the extension API nor the raw API uses these URLs?

Comment (lguychard) [reply shared/lsp/integration.test.ts]: Oops, missed this one. I updated all other URLs in tests to `git://`, see below.

Comment (efritz) [reply shared/lsp/integration.test.ts]: Would you mind updating the tests? I'm not exactly sure where the mismatch is.

Comment (lguychard) [reply shared/lsp/integration.test.ts]: https://github.com/sourcegraph/code-intel-extensions/pull/281

**Pull request merged**